test/instrumentation/modules/aws-sdk/dynamodb.js

```diff
@@ -0,0 +1,267 @@
+const agent = require('../../../..').start({
```

Comment:
Objections to moving to "$name.test.js" for new test files? That was going to be the mechanism I was going to suggest if/when we switch to node-tap to enable using its discovery via `tap test/**/*.test.js` and be able to avoid the util files like `_mock_http_client.js` and others.

As a minor utility, I find it helpful when an editor tab has ".test." in the name to differentiate from, e.g., "dynamodb.js" over in the "lib/..." tree.

Reply:
I'd rather we tackle that one as a stand alone task -- otherwise it feels like a recipe for having two sorts of tests in the repo (those with `.test.` and those without) for an indefinite  period of time. Opened an issue to discuss further: https://github.com/elastic/apm-agent-nodejs/issues/2137